### PR TITLE
Add in location name and breadcrumbs

### LINKED ISSUE #32: Show location info and breadcrumbs
Currently we see this:
image

But we should see the actual bread crumbs and names of places.

### PR DESCRIPTION
**Pull request by pbeshai**
image

Resolves #32

### CODE REVIEW

**Comment by pbeshai** [comment]
ping @vlandham

File: src/api/api.js [hunk 1]
```diff
@@ -4,13 +4,25 @@ import {
...
 // -------------
 
 /**
+ * Get information for a location
+ *
+ * @param {String} locationId The location to query (e.g., nauswaseattle)
+ * @return {Promise} A promise after the get request was made
+ */
+export function getLocationInfo(locationId) {
+  return get(`/locations/${locationId}`)
```
Comment (vlandham) [inline src/api/api.js]: should we be using `/locations/{id}/info` ?

Comment (pbeshai) [inline src/api/api.js]: Oh I didn't see that one. API docs seems to suggest it is identical to this, but you guys have liked ending with some descriptor so I can switch.

File: src/api/transforms.js [hunk 2]
```diff
@@ -160,3 +160,45 @@ export function transformSearchResults(body) {
...
+ * @param {Object} body The response body
+ * @return {Object} The transformed response body
+ */
+export function transformLocationInfo(body) {
+  // NOTE: modifying body directly means it modifies what is stored in the API cache
+  if (body.meta) {
+    const { meta } = body;
+    delete meta.parent_location_key;
+
+    meta.name = meta.client_city || meta.client_region || meta.client_country || meta.client_continent;
```
Comment (vlandham) [inline src/api/transforms.js]: we don't need a `id` field here for anything right?

**Comment by vlandham** [comment]
+1 i'm excited about breadcrumbs. Though from [Designing Interfaces](https://www.amazon.com/Designing-Interfaces-Tidwell-Jenifer-Paperback/dp/B008AUBWBI/ref=sr_1_1?ie=UTF8&qid=1472655925&sr=8-1&keywords=interface+o%27reily) i realize the name to describe this feature - "breadcrumbs" is completely wrong.

Breadcrumbs are used in Hanzel and Gretel to find their way back out of the woods. It is a metaphor indicating the recovery of a lost trail - a history.

But what they _really_ do is orient the user to where they are in the virtual world of the site. They are waypoints, or mile markers.

**Comment by pbeshai** [comment]
Thanks Uncle Jim.

**Pull request merged**